packages/grid/src/elements/pane/components/Splitter.tsx

```diff
@@ -55,6 +56,14 @@ const SplitterComponent = forwardRef<HTMLDivElement, ISplitterProps>(
       ? paneProviderContext.pixelsPerFr[orientationToDimension[orientation!]]
       : 0;
 
+    const value = isRow
+      ? paneProviderContext && paneProviderContext.getRowValue(layoutKey, true)
+      : paneProviderContext && paneProviderContext.getColumnValue(layoutKey, true);
+
+    const valueInFr = isRow
+      ? paneProviderContext && paneProviderContext.getRowValue(layoutKey)
+      : paneProviderContext && paneProviderContext.getColumnValue(layoutKey);
```

Comment:
[nitpick] for code readability but non-blocking

```suggestion
    const value = isRow
      ? paneProviderContext!.getRowValue(layoutKey, true)
      : paneProviderContext!.getColumnValue(layoutKey, true);

    const valueInFr = isRow
      ? paneProviderContext!.getRowValue(layoutKey)
      : paneProviderContext!.getColumnValue(layoutKey);
```

Reply:
These will need to be `?` notation. Updating...

Reply:
Ahh right good catch, had a dev moment there.